client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -253,12 +253,19 @@ const InvestigationTable: React.FC = (): JSX.Element => {
             }
         }
 
+    useEffect(() => {
+        window.addEventListener("keydown", handleEscKey);
+        return () => {
+            window.removeEventListener("keydown", handleEscKey);
+        };
+    }, []);
+
+    const handleEscKey = (e: KeyboardEvent) => {
+        e.key === 'Escape' && closeDropdowns()
+    }
+
     return (
-        <div tabIndex={0}
-            onClick={closeDropdowns}
-            onKeyDown={(event: React.KeyboardEvent<HTMLDivElement>) =>
-                event.key === 'Escape' && closeDropdowns()}
```

Comment:
What is the issue with the keydown event?

Reply:
the issue is that for the keydown event to work the div need's to be focused and because usually what's being focused in the dropdown the keydown doesn't register - so for it to work properly it needs to be focused somehow ( like tabIndex={0} _for example_ ) ) 
which is what caused the ugly black border side effect in the first place